### PR TITLE
Update keywords for code maintainability

### PR DESCRIPTION
**Pull request by UrsaStutsman**

### CODE REVIEW

**Review by dmohs — APPROVED** [review]
👍 👍 👍  Thank you for following-through with this!

File: src/cljs/main/broadfcui/page/workspace/data/copy_data_workspaces.cljs [hunk 1]
```diff
@@ -41,7 +41,7 @@
                                                   (get-in ws [:workspace :name]))
                                        :onClick #((:pop-to-depth props) 3)
                                        :selected-workspace ws}))
-            :toolbar-items (when-let [num-filtered (:num-filtered @state)]
+            :get-toolbar-items (when-let [num-filtered (:num-filtered @state)]
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspace/data/copy_data_workspaces.cljs]: This looks like it needs a `constantly`

File: src/cljs/main/broadfcui/page/workspace/method_configs/import_config.cljs [hunk 2]
```diff
@@ -138,7 +138,7 @@
                           (push-page {:breadcrumb-text (id->str id)
                                       :component [WorkspaceConfigChooser
                                                   (assoc props :chosen-workspace-id id)]})))
-                      :toolbar-items
+                      :get-toolbar-items
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspace/method_configs/import_config.cljs]: Same

File: src/cljs/main/broadfcui/page/workspace/method_configs/tab.cljs [hunk 3]
```diff
@@ -35,7 +35,7 @@
                                               :href (nav/get-link :workspace-method-config
                                                                   (:workspace-id props)
                                                                   (ws-common/config->id config))}))
-           :toolbar-items
+           :get-toolbar-items
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspace/method_configs/tab.cljs]: Same

File: src/cljs/main/broadfcui/page/workspace/workspace_common.cljs [hunk 4]
```diff
@@ -36,14 +36,14 @@
             {:header "Authorization Domain" :starting-width 150
              :column-data (comp :membersGroupName :authorizationDomain :workspace)
              :render #(or % "None")}]}
-    :toolbar {:items (constantly toolbar-items)}}])
+    :toolbar {:get-items (constantly get-toolbar-items)}}])
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspace/workspace_common.cljs]: This should be `:get-items (constantly toolbar-items)`

File: src/cljs/main/broadfcui/page/workspace/workspace_common.cljs [hunk 5]
```diff
@@ -59,4 +59,4 @@
                       :column-data (comp (juxt :methodNamespace :methodName :methodVersion) :methodRepoMethod)
                       :as-text (partial clojure.string/join "/")
                       :render (partial apply style/render-entity)}]}
-    :toolbar {:items (constantly toolbar-items)}}])
+    :toolbar {:get-items (constantly get-toolbar-items)}}])
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspace/workspace_common.cljs]: Same as above

**Review by davidangb — APPROVED** [review]
lgtm, modulo matt's comments. Did not test.

**Pull request merged**